lib/jsdom/living/events/EventTarget.idl

```diff
@@ -1,6 +1,12 @@
 [Exposed=(Window,Worker)]
 interface EventTarget {
-  void addEventListener(DOMString type, EventListener? callback, optional boolean capture = false);
-  void removeEventListener(DOMString type, EventListener? callback, optional boolean capture = false);
+  void addEventListener(DOMString type, EventListener? callback, optional Options options);
```

Comment:
Good, then yeah the rest of my code review comments (which are essentially about better implementing the bindings manually) still hold.